### PR TITLE
chore: replace event placeholder in venue page

### PR DESCRIPTION
**Pull request by Sboonny**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

----

- replace placeholders with chapter's event at `/dashboard/venues/{id}`

image

### CODE REVIEW

**Review by gikf — COMMENTED** [review]
I believe the displayed events are not necessarily hosted at the certain venue, but they are all organized by the _venue's_ chapter.

**Comment by Sboonny** [comment]
True, I was worried it will be indistinguishable from other placeholders, but that didn't mean I should give false info. 

I will update it tomorrow morning 👍

File: client/src/modules/dashboard/Venues/pages/VenuePage.tsx [hunk 1]
```diff
@@ -32,15 +34,59 @@ export const VenuePage: NextPage = () => {
...
             {data.venue.name}
           </Heading>
 
           <Text>{getLocationString(data.venue, true)}</Text>
           <Text>{data.venue.chapter.name}</Text>
         </ProgressCardContent>
       </Card>
-      <h3>Placeholder for events...</h3>
+      <Heading as={'h2'} marginBlock={'1em'} fontSize={'lg'}>
+        Hosted Events
```
Comment (Sboonny) [inline client/src/modules/dashboard/Venues/pages/VenuePage.tsx:46]:
```suggestion
Organized By The Venue's Chapter

File: client/src/modules/dashboard/Venues/pages/VenuePage.tsx [hunk 2]
```diff
@@ -32,15 +34,59 @@ export const VenuePage: NextPage = () => {
...
             {data.venue.name}
           </Heading>
 
           <Text>{getLocationString(data.venue, true)}</Text>
           <Text>{data.venue.chapter.name}</Text>
         </ProgressCardContent>
       </Card>
-      <h3>Placeholder for events...</h3>
+      <Heading as={'h2'} marginBlock={'1em'} fontSize={'lg'}>
+        Organized By The Venue's Chapter
```
Comment (Sboonny) [inline client/src/modules/dashboard/Venues/pages/VenuePage.tsx:46]:
```suggestion
        Organized By The Venue&apos;s Chapter

File: server/src/controllers/Venue/resolver.ts [hunk 3]
```diff
@@ -6,12 +6,34 @@ import { Venue } from '../../graphql-types';
 import { prisma } from '../../prisma';
 import { CreateVenueInputs, UpdateVenueInputs } from './inputs';
 
+const VenuesData = {
```
Comment (ojeytonwilliams) [inline server/src/controllers/Venue/resolver.ts:9]:
```suggestion
const venueIncludes = {
```
For consistency with the rest of the codebase.

File: server/src/controllers/Venue/resolver.ts [hunk 4]
```diff
@@ -6,12 +6,34 @@ import { Venue } from '../../graphql-types';
...
   venues(): Promise<Venue[]> {
     return prisma.venues.findMany({
-      include: { chapter: true },
+      include: {
+        chapter: {
+          include: {
+            events: { include: { tags: { include: { tag: true } } } },
+          },
+        },
+      },
```
Comment (ojeytonwilliams) [inline server/src/controllers/Venue/resolver.ts:36]:
```suggestion
      include: venueIncludes,
```

File: server/src/controllers/Venue/resolver.ts [hunk 5]
```diff
@@ -46,7 +68,7 @@ export class VenueResolver {
     };
     return prisma.venues.create({
       data: venueData,
-      include: { chapter: true },
+      include: VenuesData,
```
Comment (ojeytonwilliams) [inline server/src/controllers/Venue/resolver.ts:71]:
```suggestion
      include: venueIncludes,
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Review by ojeytonwilliams — APPROVED** [review]
Thanks, @Sboonny, this page is a lot less bare now. LGTM :+1:

**Pull request merged**